D2Go/create_d2go.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+
+import contextlib
+import copy
+import os
+import unittest
+from PIL import Image
+
+import torch
+from d2go.export.api import convert_and_export_predictor
+from d2go.export.d2_meta_arch import patch_d2_meta_arch
+from d2go.runner import create_runner, GeneralizedRCNNRunner
+from d2go.model_zoo import model_zoo
+
+from mobile_cv.common.misc.file_utils import make_temp_directory
+from d2go.tests.data_loader_helper import LocalImageGenerator, register_toy_dataset
+
+
+patch_d2_meta_arch()
+
+
+@contextlib.contextmanager
+def create_fake_detection_data_loader(height, width, is_train):
+    with make_temp_directory("detectron2go_tmp_dataset") as dataset_dir:
+        runner = create_runner("d2go.runner.GeneralizedRCNNRunner")
+        cfg = runner.get_default_cfg()
+        cfg.DATASETS.TRAIN = ["default_dataset_train"]
+        cfg.DATASETS.TEST = ["default_dataset_test"]
+
+        with make_temp_directory("detectron2go_tmp_dataset") as dataset_dir:
+            image_dir = os.path.join(dataset_dir, "images")
+            os.makedirs(image_dir)
+            image_generator = LocalImageGenerator(image_dir, width=width, height=height)
+
+            if is_train:
+                with register_toy_dataset(
+                    "default_dataset_train", image_generator, num_images=3
+                ):
+                    train_loader = runner.build_detection_train_loader(cfg)
+                    yield train_loader
+            else:
+                with register_toy_dataset(
+                    "default_dataset_test", image_generator, num_images=3
+                ):
+                    test_loader = runner.build_detection_test_loader(
+                        cfg, dataset_name="default_dataset_test"
+                    )
+                    yield test_loader
+
+def test_export_torchvision_format():
+    cfg_name = 'faster_rcnn_fbnetv3a_dsmask_C4.yaml'
+    pytorch_model = model_zoo.get(cfg_name, trained=True)
+
+    from typing import List, Dict
+    class Wrapper(torch.nn.Module):
+        def __init__(self, model):
+            super().__init__()
+            self.model = model
+            coco_idx_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25,
```

Comment:
Can we add a comment here mentioning that we use this to convert between the d2 output labels to the torchvision ones?